andela/taps-client
# #161319351: update request reducer and action

#### What does this PR do?
- updates the requestReducer to handle FETCH_REQUESTS
- adds action to FETCH_REQUESTS
- adds tests for the request actions
- ensures that all added tests are passing


#### Description of Task to be completed?
- the tests for the request action is 

#### How should this be manually tested?
- `git pull`
- `npm run dev`
- you should see the store requests has `loadedRequests` in it

#### Any background context you want to provide?
- the main aim of this `PR` is to make these actions readily available for collaborators who would want to use it

#### What are the relevant pivotal tracker stories?
[#161319351](https://www.pivotaltracker.com/story/show/161319351)

#### Screenshots or gifs (if appropriate)
- none

#### Questions:
- none
